conf/ocs_basic_install.yml

```diff
@@ -5,6 +5,6 @@ ENV_DATA:
   # take a look at deployment templates variables, here you is the place you
   # can overwrite all the vars you want.
   platform: 'aws'
-  worker_replicas: 6  # here you can define diff number of workers
+  worker_replicas: 3  # here you can define diff number of workers
```

Comment:
make sense to change to 3 for now but is that default worker or recommended worker/master nodes coming from doc ?

Reply:
Yes, 3 worker nodes the minimum supported configuration for OCS.

Reply:
Why do we need to change this value when this is an optional config to be passed using `--cluster-conf`? By default, we already configure 3 worker replicas. See: https://github.com/red-hat-storage/ocs-ci/blob/master/templates/install-config.yaml.j2#L6

Reply:
By default, without making any changes to the template file, the installer uses 6 worker nodes. 

1) This is not necessary for regular functional tests
2) These are AWS machines and directly increase our op-ex.

Reply:
Are you passing this file when you call `pytest`? Like `--cluster-conf conf/ocs_basic_install.yml`?

Reply:
Yes.

Reply:
Right, so what I mean is that if you want to use the defaults in our template, you do not need to pass this to pytest. If you omit that, you should only see 3 workers. This file was created to provide a way to override the defaults and as such probably shouldn't be changed to match the defaults.

Reply:
@clacroix12 I am merging this so that we can avoid the misuse when folks add additional ENVs and use that as option, I see couple in aws with 6 workers.